### PR TITLE
Matched _frameGCNcc.c and started buildtev.c

### PR DESCRIPTION
**Pull request by Yanis42**
these aren't matched yet on oot-gc but it works well on vc so here it is

### CODE REVIEW

File: src/emulator/_frameGCNcc.c [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+//     "COMBINED",    "TEXEL0",      "TEXEL1",         "PRIMITIVE",    "SHADE",
+//     "ENVIRONMENT", "1",           "COMBINED_ALPHA", "TEXEL0_ALPHA", "TEXEL1_ALPHA",
+//     "PRIM_ALPHA",  "SHADE_ALPHA", "ENV_ALPHA",      "LOD_FRAC",     "PRIM_LOD_FRAC",
+//     "0",
+// };
+
+// static char* sAlphaNames[2][8] = {
+//     {"LOD_FRAC", "TEXEL0", "TEXEL1", "PRIMITIVE", "SHADE", "ENVIRONMENT", "1", "0"},
+//     {"COMBINED", "TEXEL0", "TEXEL1", "PRIMITIVE", "SHADE", "ENVIRONMENT", "1", "0"},
+// };
```
Comment (cadmic) [inline src/emulator/_frameGCNcc.c:53]: maybe clean these up if we know they're not present any more?

File: src/emulator/_frameGCNcc.c [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    }
+
+    pFrame->aMode[FMT_OTHER1] = (pFrame->aMode[FMT_OTHER1] >> 0x10) & 1;
+    pFrame->nMode = ((pFrame->nMode & 0x40000000) & ~1) | (pFrame->aMode[3] & 1);
+    // pFrame->aMode[FMT_COMBINE_ALPHA2] = (pFrame->aMode[FMT_COMBINE_ALPHA2] & 0x40000000 & ~1) | (pFrame->aMode[3] &
+    // 1);
+    // ((u32) pFrame->aMode[5] >> 0x10) & 1, (pFrame->unkB0 & 0x40000000 & ~1) | (pFrame->aMode[3] & 1),
+    // pFrame->aMode[3])
+
+    ctP = BuildCombineModeTev(tempColor1, tempAlpha1, tempColor2, tempAlpha2, numCycles);
```
Comment (cadmic) [inline src/emulator/_frameGCNcc.c:240]: seems like BuildCombineModeTev takes some more arguments now: https://decomp.me/scratch/CMuiX

**Comment by Yanis42** [comment]
I tried to match ``SetTableTevStages``, looks like it's similar to [the gamecube version](https://decomp.me/scratch/6gwRz): https://decomp.me/scratch/RJYIB

**Comment by cadmic** [comment]
https://decomp.me/scratch/ywpou got rid of some of the fake matches from the GC version

**Review by cadmic — APPROVED** [review]

**Pull request merged**